packages/p2p/src/components/orders/order-table-row.jsx

```diff
@@ -62,51 +70,150 @@ const OrderRowComponent = observer(({ data: order, style }) => {
+                    <Text color='less-prominent' as='p' size='xxs'>
+                        {order_purchase_datetime}
+                    </Text>
+                </div>
+            </React.Fragment>
+        );
+    };
     return (
-        <div onClick={() => order_store.setQueryDetails(order)} style={style}>
-            <Table.Row
-                className={classNames('orders__table-row orders__table-grid', {
-                    'orders__table-grid--active': general_store.is_active_tab,
-                    'orders__table-row--attention': !isOrderSeen(id),
-                })}
-            >
-                <Table.Cell>
-                    {(is_buy_order && !is_my_ad) || (is_sell_order && is_my_ad) ? (
-                        <Localize i18n_default_text='Buy' />
-                    ) : (
-                        <Localize i18n_default_text='Sell' />
-                    )}
-                </Table.Cell>
-                <Table.Cell>{id}</Table.Cell>
-                <Table.Cell>{other_user_details.name}</Table.Cell>
-                <Table.Cell>
-                    <div
-                        className={classNames('orders__table-status', {
-                            'orders__table-status--danger': should_highlight_danger,
-                            'orders__table-status--alert': should_highlight_alert,
-                            'orders__table-status--success': should_highlight_success,
-                            'orders__table-status--disabled': should_highlight_disabled,
+        <React.Fragment>
+            {isMobile() ? (
+                <div
+                    style={style}
+                    className={classNames('orders__expander', {
+                        'orders__expander-heighlight-danger': should_highlight_danger,
+                    })}
+                >
+                    <div className='orders__expander-top' onClick={() => order_store.setQueryDetails(order)}>
+                        <Table.Row className='orders__expander-header'>
+                            <Table.Cell
+                                className={classNames('orders__table-row orders__expander-header', {
+                                    'orders__table-grid--active': is_active,
+                                })}
+                            >
+                                <div
+                                    className={classNames('orders__expander-status', {
+                                        'orders__table-status--danger': should_highlight_danger,
+                                        'orders__table-status--alert': should_highlight_alert,
+                                        'orders__table-status--success': should_highlight_success,
+                                        'orders__table-status--disabled': should_highlight_disabled,
+                                    })}
+                                >
+                                    {status_string}
+                                </div>
```

Comment:
Since the status classes are the same for both browser and responsive, can we use one?

Reply:
I believe, creating a new component for this status is not necessary.